test/acceptance/schema/update_class_test.go

```diff
@@ -68,3 +68,117 @@ func TestUpdateClassDescription(t *testing.T) {
 		assert.Equal(t, res.Payload.Description, newDescription)
 	})
 }
+
+func TestUpdatePropertyDescription(t *testing.T) {
+	className := "C2"
+	propName := "p1"
+	nestedPropName := "np1"
+
+	delete := func() {
+		params := clschema.NewSchemaObjectsDeleteParams().WithClassName(className)
+		_, err := helper.Client(t).Schema.SchemaObjectsDelete(params, nil)
+		assert.Nil(t, err)
+	}
+	defer delete()
+
+	t.Run("delete class if exists", func(t *testing.T) {
+		delete()
+	})
+
+	t.Run("initially creating the class", func(t *testing.T) {
```

Comment:
why not move the code of this .Run and the one above outside? Then each of the real tests can run independently without running the completely file